tools/devtool

```diff
@@ -486,7 +486,7 @@ cmd_build() {
 }
 
 function cmd_make_release {
-    cmd_test -- --reruns 2 || die "Tests failed!"
+    cmd_test -- --reruns 2 -n 8 || die "Tests failed!"
```

Comment:
When this parallelization is enabled, does the test work well?
I'm curious about why we haven't parallelize this so far.